app/models/booking_response.rb

```diff
@@ -5,7 +5,7 @@ class BookingResponse
 
   attribute :visit
 
-  attribute :selection, Integer
+  attribute :selection, String
```

Comment:
My obsession with StringInquirer continues.. You <i>could</i> do something like this..

```ruby
class StringWithInquiry < Virtus::Attribute
  def coerce(value)
    value.to_s.inquiry
  end
end

class BookedResponse
  attribute :selection, StringWithInquiry
  delegate :no_allowance?, :visitor_not_on_list?, :visitor_banned?, to: :selection
end
```

This might be useful/reusable for other places in the app that use `inquiry` on strings values.